### DISCUSSION TITLE
Reserve rooms

### DISCUSSION THREAD

**Discussion by aleamar264** [kuzmoyev/google-calendar-simple-api#171] [Q&A]
Hi, I want to use this API to reserve some rooms in the company, do you know if this API have the ability to do that ?

**Comment by kuzmoyev** [comment]
How do you reserve the rooms normally via UI? Do you add room as an attendee?

**Reply by aleamar264** [reply]
when we are on personal account, only appear location, but an organization we have two options, location and room 
![image](https://github.com/kuzmoyev/google-calendar-simple-api/assets/28543487/91bf93e2-db72-4e8d-8ec6-80aec80b2fc9)
it's on the same page that location

**Reply by aleamar264** [reply]
So I don't know if we can trait this like a 'location'

**Reply by kuzmoyev** [reply]
Try adding the event with the room manually via UI then retrieve the event using the gcsa and check the `.location` field.

**Reply by aleamar264** [reply]
thanks, I will try that. I also was looking on the documentation and nothing about room. If I have succeed with that lyk
